.travis.install-elasticsearch.sh

```diff
@@ -17,6 +19,16 @@ function install_from_targz {
     sudo mv $2 $INSTALL_PATH
   fi
   /opt/es/$2/bin/elasticsearch -Vv
+  remove_if_erroneus $2
+}
+
+function remove_if_erroneus {
+  if [ ! -f "/opt/es/$1/bin/elasticsearch" ]; then
+    ls -l /opt/es/
+    ls -l /opt/es/$1
+    ls -l /opt/es/$1/bin
+    rm -r /opt/es/$1
```

Comment:
is the `-f` flag missing? Also you can try to use`tree` command to list all files that will get removed and print a message about it.

Reply:
force isn't needed here. But the -v will be helpful